### PR TITLE
 Printing numbers from 0 to the number given by the user

### LINKED ISSUE #1: Task description
precision-of-numbers

### PR DESCRIPTION
**Pull request by wojciechkostecki**
Closes #1

### CODE REVIEW

**Review by rbobko93 — CHANGES_REQUESTED** [review]
You should be starting printing from 0 and you are not. You are printing starting with 0.1 or -0.1

File: src/Main.java [hunk 1]
```diff
@@ -11,17 +11,18 @@ public static void printNumbers() {
         int number = scanner.nextInt();
 
         if (number > 0) {
-            double x = 0;
-            while (x < number) {
-                x += 0.1;
+            double x=0;
+            while (x < number + 0.1) {
```
Comment (rbobko93) [inline src/Main.java:15]: Why condition is x < number + 0.1 ? Why number + 0.1 ? Why you don't have number - 0.1 as well?

File: src/Main.java [hunk 2]
```diff
@@ -11,17 +11,18 @@ public static void printNumbers() {
...
                 System.out.printf("%.1f" + " ", x);
+                x += 0.1;
             }
         }
         if (number < 0) {
             double y = 0;
+            System.out.print(0 + " ");
```
Comment (rbobko93) [inline src/Main.java]: Don't even try this solution with me. Remove it.

**Review by rbobko93 — CHANGES_REQUESTED** [review]

File: src/Main.java (hunk 1 above)
Comment (rbobko93) [reply src/Main.java:15]: @wojciechkostecki Why did you resolve this conversation without responding to it? You didn't change anything in the code.

Comment (wojciechkostecki) [reply src/Main.java:15]: I added below -0.1.

Comment (rbobko93) [reply src/Main.java:15]: I know. I can see, but this answers only one of the questions.

**Review by rbobko93 — APPROVED** [review]

**Pull request merged**